### PR TITLE
[IMP] Add a notice to step 9

### PR DESCRIPTION
**Pull request by NL66278**
There are two deleteTask functions in step 9. This will make sure the developer/student
does not miss one of those.

### CODE REVIEW

File: doc/learning/tutorial_todoapp.md [hunk 1]
```diff
@@ -502,6 +502,10 @@ deleteTask(task) {
 Notice that the `onDelete` prop is defined with a `.bind` suffix: this is a special
 suffix that makes sure the function callback is bound to the component.
 
+Notice also that we have two functions named `deleteTask`. The one in the Task
+component just delegates the work to the Root component that owns the task list
+via the `onDelete' property.
```
Comment (caburj) [inline doc/learning/tutorial_todoapp.md:507]: Suggestion: Wrap the onDelete with 2 backticks.

\`onDelete' -> \`onDelete\`

Comment (NL66278) [reply doc/learning/tutorial_todoapp.md]: @caburj Sorry for the typo! I amended the commit to correct it.

Comment (caburj) [reply doc/learning/tutorial_todoapp.md]: @NL66278 One more thing. The commit title, can you change it to "[DOC] Add a notice to step 9". Then we can merge. :)

**Comment by caburj** [comment]
And one thing we might have forgotten. Can you sign the CLA if you haven't already? Here's the link for the info: https://github.com/odoo/odoo/blob/master/doc/cla/sign-cla.md

Pardon the extra formality.

**Comment by NL66278** [comment]
@caburj  I changed the commit message and the CLA signature is here: https://github.com/odoo/odoo/blob/master/doc/cla/corporate/therp.md?plain=1#L24

**Review by caburj — APPROVED** [review]

**Pull request merged**